src/Azure/Orleans.Clustering.AzureStorage/MultiClusterNetwork/GossipTableInstanceManager.cs

```diff
@@ -142,14 +144,14 @@ await instance.storage.InitTableAsync()
             {
                 string errorMsg = String.Format("Unable to create or connect to the Azure table {0} in {1}", 
                     instance.TableName, initTimeout);
-                instance.logger.Error((int)AzureUtils.Utilities.ErrorCode.AzureTable_32, errorMsg, te);
+                instance.logger.Error((int)TableStorageErrorCode.AzureTable_65, errorMsg, te);
```

Comment:
Why did we changed from 32 to 65 here? And below we changed from 33 to 61. But the value in the catch didn't changed.

Reply:
Ok 32 and 33 were already used in the `GossipTableInstanceManager`, I suppose that's why Attilah changed them to 61 and 62?

@sergeybykov is it ok to change the error code here? Should we document this somewhere?